### DISCUSSION TITLE
How to enable light probes and reflection probes on character?

### DISCUSSION THREAD

**Discussion by rebeccakzpt** [naverz/zepeto-studio-global#5] [Q&A]
The lighting in my world is complicated. It's impossible to use real-time lighting for all light sources. But the character doesn't have light probes and reflection probes turned on by default. It can be too dark. Is there a way to turn them on?

**Comment by jhslee** [comment]
Hello @rebeccakzpt I'm the Global Tech Evangelist 👋 . I apologize that your question has been left so long unanswered 🥲 

I see your question has been answered [here](https://github.com/naverz/zepeto-studio-kor/discussions/739)
If anyone else is in need of an answer for light/reflection probes, please refer to the discussion thread above~

Thank you!
